lib/blocs/request_manager/offline_request_manager.dart

```diff
@@ -66,6 +66,18 @@ class OfflineRequestManager implements RequestManager {
     }());
   }
 
+  CancelableOperation<Plan> fetchBikePlan(
+    BuildContext context,
+    TrufiLocation from,
+    TrufiLocation to,
+  ) {
+    return CancelableOperation.fromFuture(() async {
+      throw FetchOfflineRequestException(
```

Comment:
Would it not be better to throw the correct type
```suggestion
      throw UnimplementedError("Fetch plan as bike route offline is not implemented yet.");
```